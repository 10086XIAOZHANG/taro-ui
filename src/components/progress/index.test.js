import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'

import AtProgress from '../../../.temp/components/activity-indicator/index'

describe('Progress Snap', () => {
  it('render initial Progress', () => {
    const componet = renderToString(<AtProgress />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props percent', () => {
    const componet = renderToString(<AtProgress percent={25} />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props isHidePercent', () => {
    const componet = renderToString(<AtProgress isHidePercent />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props strokeWidth ', () => {
    const componet = renderToString(<AtProgress strokeWidth={10} />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props color ', () => {
    const componet = renderToString(<AtProgress color='#FF4949' />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props status(progress) ', () => {
    const componet = renderToString(<AtProgress status='progress' />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props status(error) ', () => {
    const componet = renderToString(<AtProgress status='error' />)
    expect(componet).toMatchSnapshot()
  })

  it('render Progress -- props status(success) ', () => {
    const componet = renderToString(<AtProgress status='success' />)
    expect(componet).toMatchSnapshot()
  })
})
