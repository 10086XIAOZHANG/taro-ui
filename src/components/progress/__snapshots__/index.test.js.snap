// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Snap render Progress -- props color  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#FF4949;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props isHidePercent 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props percent 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props status(error)  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props status(progress)  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props status(success)  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render Progress -- props strokeWidth  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`Progress Snap render initial Progress 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;
